.circleci/config.yml

```diff
@@ -939,8 +939,8 @@ jobs:
         type: integer
         default: 0
       nodejs_version:
-        type: integer
-        default: 14
+        type: string
+        default: latest
```

Comment:
Which other task uses this with default?

Reply:
In this PR only `t_ems_compile_ext_zeppelin`. But I have a bunch of other PRs updating external tests. Eventually most of the `t_ems_compile_XYZ` jobs will be using the default because they work fine on nodejs 17. And I would use it for `t_ems_test_XYZ` jobs too but most of them now use Hardhat and it does not support node 17 yet (https://github.com/nomiclabs/hardhat/issues/1988).